ZodiacBuddy/BonusLight/BonusLightManager.cs

```diff
@@ -79,10 +79,10 @@ public void UpdateLightBonus(uint? territoryId, DateTime? date, string? message)
             var mainhand = Util.GetEquippedItem(0);
             var offhand = Util.GetEquippedItem(1);
 
-            if (NovusRelic.Items.ContainsKey(mainhand.ItemID) ||
-                NovusRelic.Items.ContainsKey(offhand.ItemID) ||
-                BraveRelic.Items.ContainsKey(mainhand.ItemID) ||
-                BraveRelic.Items.ContainsKey(offhand.ItemID))
+            if (!NovusRelic.Items.ContainsKey(mainhand.ItemID) &&
+                !NovusRelic.Items.ContainsKey(offhand.ItemID) &&
+                !BraveRelic.Items.ContainsKey(mainhand.ItemID) &&
+                !BraveRelic.Items.ContainsKey(offhand.ItemID))
```

Comment:
Oof, getting the check completely backwards.

Reply:
at this point, ive got to farm up another set of brave books for testing. so definite oof.